src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -336,8 +336,9 @@ private void QueryTextBox_TextChanged(object sender, TextChangedEventArgs e)
         {
             var textBox = (TextBox)sender;
             var text = textBox.Text;
+            var autoCompleteText = SearchBox.AutoCompleteTextBlock.Text;
 
-            if (string.IsNullOrEmpty(text))
+            if (string.IsNullOrEmpty(text) || autoCompleteText.IndexOf(text, StringComparison.InvariantCulture) != 0)
```

Comment:
Is `InvariantCulture` the correct one to use here? The two strings should be an exact match so wouldn't Ordinal make more sense here? Otherwise you could end up with two strings that vary a little overlapping which would appear as a rendering bug

Reply:
I agree this would have a shortcoming of getting a match in case of character expansion as mentioned here: https://stackoverflow.com/a/20085219. I will update it to the ordinal match.